webapp/src/pages/LoginPage/LoginPage.tsx

```diff
@@ -0,0 +1,144 @@
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+import {
+  Button,
+  ButtonVariant,
+  IconVariant,
+  Input,
+  InputVariant,
+  Text,
+  TextColor,
+  TextVariant,
+  TextWeight,
+} from '../../shared/components';
+import { ResponseError } from '../../shared/generated';
+import { authApi } from '../../shared/services/ApiService';
+import { LOGIN_EP_URL, REGISTER_URL, USERS_URL } from '../../shared/urls';
+import './LoginPage.css';
+
+type SubmitStatus = {
+  type: 'success' | 'error' | 'pending';
+  message: string;
+};
+
+export default function LoginPage() {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  const [status, setStatus] = useState<SubmitStatus>({
+    type: 'pending',
+    message: '',
+  });
+
+  async function login() {
+    try {
+      await authApi.authControllerLoginLocalUser({
+        loginUserDto: { username, password },
+      });
+      window.location.href = USERS_URL;
```

Comment:
what was the problem with navigate?

Reply:
For some reason it navigates to the landing page and not to the users page, and if you refresh then it do goes to the users page